### PR TITLE
Added new TCs for dedicated bearer testing

### PR DESCRIPTION
**Pull request by pruthvihebbani**
1. Added 2 new testcases to test dedicated bearer
2. PR https://github.com/facebookincubator/magma/pull/651 needs to be pulled before executing these TCs as it has some bug fixes

### CODE REVIEW

File: lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_multi_ue.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    def setUp(self):
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+        self._spgw_util = SpgwUtil()
+
+    def tearDown(self):
+        self._s1ap_wrapper.cleanup()
+
+    def test_attach_detach(self):
+        """ attach/detach + dedicated bearer test with 32 UEs """
+        num_ues = 32
```
Comment (talkhasib) [inline lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_multi_ue.py]: 32 is too many
Lets use 4 UEs in this test

File: lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_multi_ue.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        self._s1ap_wrapper.configUEDevice(num_ues)
+
+        for i in range(num_ues):
```
Comment (talkhasib) [inline lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_multi_ue.py:34]: Change this test to 
1- Attach all UEs
2- Once all UEs attach, add a dedicated bearer to each one of them
3- Delete dedicated bearers from all UEs
4- detach all UEs

File: lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_looped.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+        self._spgw_util = SpgwUtil()
+
+    def tearDown(self):
+        self._s1ap_wrapper.cleanup()
+
+    def test_attach_detach(self):
+        """ attach/detach + dedicated bearer test in loop with a single UE """
+        num_ues = 1
+        loop = 10
```
Comment (talkhasib) [inline lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_looped.py]: 10 is too long for the test
lets loop 3 times only

File: lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_multi_ue.py (hunk 2 above)
Comment (pruthvihebbani) [reply lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_multi_ue.py:34]: done

File: lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_multi_ue.py (hunk 1 above)
Comment (pruthvihebbani) [reply lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_multi_ue.py]: done

File: lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_multi_ue.py (hunk 2 above)
Comment (pruthvihebbani) [reply lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_multi_ue.py:34]: done

File: lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_looped.py (hunk 3 above)
Comment (pruthvihebbani) [reply lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_looped.py]: done

**Pull request closed without merging**